apps/mobile/src/screens/adoptions/adoptions.screen.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box } from 'native-base';
+import { SafeAreaView } from 'react-native-safe-area-context';
+
+import { Adoption, MyPetsList } from './compose/my-pets-list';
+
+import SafeArea from '@/components/safe-area';
+import Topbar from '@/components/topbar';
+
+const MyAdoptionsScreen = () => {
+  return (
+    <SafeArea>
+      <Box py={8} px={4}>
+        <Topbar />
+        <Box mt="6" py={8}>
+          <MyPetsList adoptions={DATA as Adoption[]} />
+        </Box>
+      </Box>
+    </SafeArea>
```

Comment:
```suggestion
    <SafeArea>
      <Delimiter flex="1">
        <Topbar />
        <Box mt="4">
          <MyPetsList adoptions={DATA as Adoption[]} />
        </Box>
      </Delimiter>
    </SafeArea>
```

use the `Delimiter` component